hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/testutils/HoodieClientTestUtils.java

```diff
@@ -322,7 +322,7 @@ public static HoodieTableMetaClient createMetaClient(JavaSparkContext jsc, Strin
    * @return a new {@link HoodieTableMetaClient} instance.
    */
   public static HoodieTableMetaClient createMetaClient(SparkSession spark, String basePath) {
-    return HoodieTestUtils.createMetaClient(spark.sessionState().newHadoopConf(), basePath);
+    return HoodieTestUtils.createMetaClient(HoodieStorageUtils.getStorageConf(spark.sessionState().newHadoopConf()), basePath);
```

Comment:
Use `HoodieTestUtils#createMetaClient(SparkSession spark, String basePath)`?

Reply:
Definitely can't have spark in HoodieTestUtils